scripts/specializations/CpAICombineUnloader.lua

```diff
@@ -162,3 +163,8 @@ function CpAICombineUnloader:replaceDriveStrategies(fieldPolygon, jobParameters)
     --- Only the last driving strategy can stop the helper, while it is running.
     table.insert(spec.driveStrategies, cpDriveStrategy)
 end
+
+--- Forces the driver to unload now.
+function CpAICombineUnloader:startCpCombineUnloaderUnloading()
+    print("Drive now!")
```

Comment:
@pvaiko This is the button callback from drive to unload (driveNow), that needs to be implemented.